packages/jupyter-ai/jupyter_ai/handlers.py

```diff
@@ -174,7 +185,7 @@ def get_chat_user(self) -> ChatUser:
         login = getpass.getuser()
         initials = login[0].capitalize()
         return ChatUser(
-            username=login,
+            username=self.current_user.username,
```

Comment:
One last thing: Can you implement the fixes for #1025 and #1027 in a separate PR? Based on the discussion in #1025, I'm comfortable with this portion of the PR being merged. Splitting this up & rebasing would also help build a more readable changelog (and make sure you receive credit for fixing multiple issues 😁).